### DISCUSSION TITLE
Server: add the ability for mooving files between channels

### DISCUSSION THREAD

**Discussion by amirmahdifard** [BearWare/TeamTalk5#1583] [Ideas]
### Description

guys, really grate to you! i just figgered out that in Qt client, we can see the files list of other channels if we are not joind in that channel, wich is really grate, and also, we can upload files to other channels too if we are not in that channels and we are in another channel: it's really grate! now that you have a grate thing as this, it's bettar to also add function of mooving files between channels to the server side, and also to the client side so it can handle it: the server side is i think would be simple, just a function to change the file path in the xml file to the new channel path, and so it would be stil downloadable and deletable in the channel as normal, and an option to the client so we can select files for moov and then select the target channel to moov to: this feecher is really important to add, because with out this feecher, if we want to moov a file to another channel, we have to delete it, and then upload it to the new channel wich will use lots of internet data depends on size of that file: so please work on this at some point: server and client vertion: 5.11

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by amirmahdifard** [comment]
guys, y you don't have any comment about this, this feecher is really grate too add, since other apps doesn't have this ability also

**Comment by lbk2907** [comment]
Add the copy function also.
